From b0ce5ea459e791eeaccb992081cfa95bdf42059c Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Tue, 23 Jan 2024 15:46:33 +0200
Subject: [PATCH] [SW-172556] gaudi2: erom : set non bis complaint mode

This patch corrects bug in erom which responds to check of
bis complaint mode. As erom is not available in bis complaint
mode, this check should always return false.

Change-Id: I794748be8f43a17e5f8e9ae1f188c5f6442a0fbf
---

diff --git a/app/ext_rom/erom_pcie.c b/app/ext_rom/erom_pcie.c
index 3583b31..c12c32a 100644
--- a/app/ext_rom/erom_pcie.c
+++ b/app/ext_rom/erom_pcie.c
@@ -233,5 +233,5 @@
  */
 bool preboot_is_bis_compliant_mode(void)
 {
-	return true;
+	return false;
 }
